src/scenes/DeliverablesRouter/index.tsx

```diff
@@ -1,13 +1,25 @@
 import React from 'react';
 import { Route, Routes } from 'react-router-dom';
 
+import ProjectProvider from 'src/providers/Project/ProjectProvider';
+import ParticipantProjectProvider from 'src/scenes/AcceleratorRouter/ParticipantProjects/ParticipantProjectProvider';
+
 import DeliverableViewWrapper from './DeliverableViewWrapper';
 import DeliverablesList from './DeliverablesList';
 
 const DeliverablesRouter = (): JSX.Element => {
   return (
     <Routes>
-      <Route path={'/:deliverableId/submissions/:projectId'} element={<DeliverableViewWrapper />} />
+      <Route
+        path={'/:deliverableId/submissions/:projectId'}
+        element={
+          <ProjectProvider>
+            <ParticipantProjectProvider>
+              <DeliverableViewWrapper />
+            </ParticipantProjectProvider>
+          </ProjectProvider>
+        }
+      />
```

Comment:
I ended up wrapping only the deliverable view route since it's the only route with `projectId` in the path, which is required by the providers.